src/components/DpInput/DpInput.vue

```diff
@@ -30,124 +29,172 @@
       :required="required"
       :autocomplete="autocomplete !== '' ? autocomplete : null"
       :size="(size && size > 0) ? size : null"
-      v-model="currentValue"
-      @blur="$emit('blur', currentValue)"
+      v-model="props.value"
```

Comment:
Do we want to use `props.value` or just `value`?

Reply:
see discussion below.